builder/job.go

```diff
@@ -46,6 +46,16 @@ var validCommitCommands = map[string]bool{
 	"workdir":    true,
 }
 
+// BuiltinAllowedBuildArgs is list of built-in allowed build args
+var BuiltinAllowedBuildArgs = map[string]bool{
+	"HTTP_PROXY":  true,
+	"HTTPS_PROXY": true,
+	"http_proxy":  true,
+	"https_proxy": true,
+	"FTP_PROXY":   true,
+	"NO_PROXY":    true,
```

Comment:
Not related to the last changes, but I keep forgetting to ask; is there a reason `FTP_PROXY` and `NO_PROXY` don't have a lowercase variant (unlike the other ones)?